build.sh

```diff
@@ -12,6 +12,16 @@ function banner1() {
     echo $'\n'"===" "$@" $'\n'
 }
 
+function checkGoVersion() {
+    local expectedVersion=`cat GOLANG_VERSION`
```

Comment:
Combining `local` with assignment via command substitution has the problem that an error in the command substitution is not handled. Split variable declaration and assignments:

    local myvar
    myvar=$(echo blah)